ocaml/xapi/xapi_vgpu_type.ml

```diff
@@ -533,6 +546,139 @@ module Intel = struct
     List.map (find_or_create ~__context) types
 end
 
+module MxGPU = struct
+  let amd_vendor_id = 0x1002
+
+  let ( *** ) = Int64.mul
+  let ( /// ) = Int64.div
+  let ( +++ ) = Int64.add
+  let ( --- ) = Int64.sub
+  let mib x = List.fold_left Int64.mul x [1024L; 1024L]
```

Comment:
Quite hard-core to use a fold here :) Also, we've just defined `***`, so why not just `x *** 1024L *** 1024L`?